lbrynet/lbrynet_daemon/LBRYDaemon.py

```diff
@@ -242,6 +243,8 @@ def __init__(self, root, wallet_type=None):
             self.session_settings['last_version'] = self.default_settings['last_version']
             self.first_run_after_update = True
             log.info("First run after update")
+            log.info("lbrynet %s --> %s" % (self.session_settings['last_version']['lbrynet'], self.default_settings['last_version']['lbrynet']))
```

Comment:
break this into multiple lines.  I generally prefer putting each argument on its own line:

```
log.info(
    "lbrynet %s --> %s",
    self.session_settings['last_version']['lbrynet'], 
    self.default_settings['last_version']['lbrynet']
)
```